setup.py

```diff
@@ -15,7 +15,7 @@
 # Parameters for build
 params = {
     'name': name,
-    'version': '2.1.13',
+    'version': '2.1.17',
```

Comment:
I guess you could name it just 2.1.14

Reply:
really no need to go to 17, I can delete the scratch build out of the testing repo for you